opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -376,6 +377,11 @@ void VFPProdTable::check() {
                     for (size_type t=0; t<m_data.shape()[0]; ++t) {
                         if (m_data[t][w][g][a][f] < bhp_last) {
                             ++num_decreasing;
```

Comment:
Of course the code will need to be adapted to the new message, but a counter is probably not what you want since all violations will be printed (instead the number of violations, namely num_decreasing).